pkg/cni/types/types.go

```diff
@@ -148,9 +151,13 @@ func (cl *confLoader) LoadConf(args *skel.CmdArgs) (*NetConf, error) {
 	}
 
 	// overlay config with defaults
+	var defaultPoolName string
+	if !fileConf.ForcePoolName {
```

Comment:
I think we should not limit this to the fileConf. We should support this option as STDIN ARG also.

Reply:
makes sense